modules/UI/UI.js

```diff
@@ -340,30 +340,30 @@ UI.start = function() {
     VideoLayout.resizeVideoArea(true, true);
 
     sharedVideoManager = new SharedVideoManager(eventEmitter);
-    // eslint-disable-next-line no-negated-condition
-    if (!interfaceConfig.filmStripOnly) {
+
+    if (interfaceConfig.filmStripOnly) {
+        $('body').addClass('filmstrip-only');
+        Filmstrip.setFilmstripOnly();
+        APP.store.dispatch(setNotificationsEnabled(false));
+    } else {
         // Initialise the recording module.
         if (config.enableRecording) {
             Recording.init(eventEmitter, config.recordingType);
         }
 
         // Initialize side panels
         SidePanels.init(eventEmitter);
-    } else {
-        $('body').addClass('filmstrip-only');
-        UI.showToolbar();
-        Filmstrip.setFilmstripOnly();
-        APP.store.dispatch(setNotificationsEnabled(false));
-    }
 
-    if (interfaceConfig.VERTICAL_FILMSTRIP) {
-        $('body').addClass('vertical-filmstrip');
-    }
+        // XXX: We only do this when not in film strip only mode, since
+        // vertical toolbars are not implemented there.
+        if (interfaceConfig.VERTICAL_FILMSTRIP) {
+            $('body').addClass('vertical-filmstrip');
+        }
```

Comment:
So this class no longer gets added when in vertical filmstrip only mode? How does that look? The idea is that even though vertical filmstrip only might not have a decent experience that it still functions. Just like horizontal filmstrip with the new toolbar...it works but it sucks but it works, and it works without much extra effort code-wise.

Reply:
What I did there is essentially not add it, yes. For all intents and purposes we only seem to support horizontal filmstrip mode.

I wonder if we should remove the setting and make it intrinsic: normal view uses a vertical filmstrip and filmstriponly uses a horizontal one.

Reply:
What I'm saying is it's somewhat supported but clearly not a great UX; it's not the prettiest but it still works like the regular vertical filmstrip. I recall being explicitly told while implementing vertical filmstrip to make vertical filmstrip display in filmstrip only mode. That it's not optimized or pretty is not the same as not being supported. As for forcing orientation, I commented in the jira ticket you filed and I'll repeat here: it was actually easier for now to add 5-10 lines of CSS to support both filmstrip orientations than to refactor web right now to force a certain filmstrip orientation. When going into horizontal toolbar it was discussed to force the filmstrip orientations, but there are a lot of tacked on conditionals spread across web that made that effort hairier than I cared for.

Reply:
So I guess what I'm getting at is removing the class would be something I'm okay with if it's a more concerted effort to kill off support for vertical filmstrip in filmstrip only mode. Removal of the styling itself I would consider insufficient.